### DISCUSSION TITLE
Not able to include language pack in builder.

### DISCUSSION THREAD

**Discussion by mobiliseapplabllp** [givanz/Vvveb#193] [Q&A]
Hi

I hope you're doing well. I have been working with the builder, and it's been an excellent tool for my project. However, I am currently facing an issue while trying to include a language pack into the builder. Despite my efforts, I haven’t been able to get it to function as expected.

Could you kindly guide me on how to correctly integrate language packs or if there is any documentation that addresses this aspect? Your assistance would be greatly appreciated.

Can you please refer me to any documentation?

Thank you for your time and support!

**Comment by givanz** [comment]
Hi

Language translations is done with gettext files, you can find the available translations here https://github.com/search?q=org%3AVvveb+language-pack&type=repositories

They can be automatically installed from the admin > languages > language https://demo.vvveb.com/admin/index.php?module=localization/languages

If you want to manually install a language pack then you can copy the language pack files to `/locale` folder, the structure is `vvveb/locale/fr_FR/LC_MESSAGES/vvveb.mo` you can translate and compile .po files to .mo using a gettext editor such as poedit.
